benchmark/Orth_speed.cc

```diff
@@ -133,10 +131,10 @@ test_speed_helper(int64_t m, int64_t n, uint32_t seed) {
     std::this_thread::sleep_for(std::chrono::seconds(5));
 
     // Stabilization Constructor
-    PLUL<T> Stab_PLU(false, false);
-    CholQRQ<T> Orth_CholQR(false, false);
-    HQRQ<T> Orth_HQR(false, false);
-    GEQR<T> Orth_GEQR(false, false);
+    RandLAPACK::PLUL<T> Stab_PLU(false, false);
+    RandLAPACK::CholQRQ<T> Orth_CholQR(false, false);
+    RandLAPACK::HQRQ<T> Orth_HQR(false, false);
+    //RandLAPACK::GEQR<T> Orth_GEQR(false, false);
```

Comment:
Why not use ifdef endif macros? Commenting out seems excessive. (@TeachRaccooon)

Reply:
Because RandLAPACK::GEQR does not actually exist anymore. I am not sure why that was not detected on linux compilations. The reason I did not remove this is to address it in future, when we get back to the Orth/Stab technical report, as this benchmark has been written for it.

Reply:
Excellent.